Shuttle (New): SWS 'Raven'

CloudOfNeutronFlux opened the pull request:
## About the PR
Adds a new shuttle to the expeditionary roster.

## Why / Balance
With all the cool ship submissions in the freeze at the moment, I noticed a lack of expedition ships and I think the Lodge could really use a new one, especially with the Gasbender and the Ambition potentially leaving the roster. This also aligned with me wanting to create an easily recognizable mass scanner signature, as Lodge ships are for the most part very Shaped vessels.

The design the shuttle's settled on in the end is a cargo-salvage runner with EVA access covering three out of four dungeon location directions, a comfortably spaceous cargo bay to store all your loot in, and an atmospheric Star Trek-esque bridge with the mandatory "Make It So" throne and a dedicated Red Alert mode button.

The ship supplies crew with EVA softsuits instead of hardsuits, primarily because I personally prefer softsuits aesthetically, but also because speed is a bit more important on expeditions until you get into higher tiered hardsuits (be it through buying from science or finding during planet missions).

The Raven is designed with a crew of four people in mind, but with rotating "shifts" and some role management from the captain, it can easily support up to seven.

## How to test
Buy the ship at the Expeditionary Lodge.

## Media
![Content Client_pj1xCH3nIy](https://github.com/user-attachments/assets/b08c3a10-be4b-4e50-9e95-3a5f5c2e6138)
![Content Client_6QIS673fUr](https://github.com/user-attachments/assets/75828ae7-9a6e-4534-91d4-b625ad2be71f)
![Content Client_2uyLJiI5OJ](https://github.com/user-attachments/assets/b923153f-a79b-4f92-8cab-902abc602de7)

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Adds SWS Raven, an expedition-capable shuttle.

dvir001 commented:
![image](https://github.com/user-attachments/assets/26132d45-da55-48f7-b2fa-c099404cce45)

Clay500z commented:
I like the design, though I feel like that hallway to the bridge is going to become too much of a chore to cross over and over. Cutting off the hallway would make it more functional. Though that would block off the dock at the bridge...

Houtblokje commented:
> I like the design, though I feel like that hallway to the bridge is going to become too much of a chore to cross over and over. Cutting off the hallway would make it more functional. Though that would block off the dock at the bridge...

They could probably make the hallway like, 3 tiles shorter. I think that would make the shape kinda better too, right now it feels like the neck of the bird is too long compared to the wings.

Temoffy commented:
Looks fun!

In addition to Dusty's comments, I have a few thoughts if you're amiable to them:

something like this might be a decent alternate thrust layout to address the wall-facing thrusters:
![image](https://github.com/user-attachments/assets/9cc5f0a2-b831-43f0-a3f9-65d63323a046)

A curtain over the sink and moving the light to the toilet might be nice for privacy.

would reinforced or plasma reinforced directional windows be out of the question in the neck? Might open up the space without changing the profile, but I do think the neck is a smidge long.

is that plating under the thrusters? Is using that instead of lattice intentional or did it just end up that way?

Looking forward to how this is refined!

CloudOfNeutronFlux commented:
Don't have the spoons to respond to each piece of feedback individually, but I've read all that was written here and tried to account for it as much as possible.

Redid the whole layout. 

Most of the issues raised in feedback stemmed from me wanting to keep engineering closer to the middle of the ship. Ditched that idea, moved engineering to one of the wing tips. Shuttle should be easier to navigate now, plus it allowed for a bigger cargo bay and equipment for an additional crew member.

Squished ~~the cat~~ the long corridor significantly and used the leftover tiles to beef it up instead, making space for a conference room.

Remade the bridge while I was shortening the neck, it should look more like a Star Trek bridge now.

Replaced navigational strobes with tube lights.

Red Alert mode now keeps Bridge, Cargo Bay and Medbay partially lit. Other rooms don't get this treatment to incentivise crew to stop drinking and start preparing.

Clay500z commented:
You're COOKING

Jakumba commented:
Pulled it and had a walk around the reworked version. It feels really comfortable and it flows nicely, the bigger cargo bay was a good choice I think.
My only notes are:

- Light posts on the nose and wingtips might be cool
- the AME shielding needs to be rejigged to be the correct shape

Otherwise looks good, the crew cabin is really nice and the smoking lounge in the "neck" is very cosy.

CloudOfNeutronFlux commented:
Post lights added to lattice on the wings and bridge. AME rebuild. Captain's quarters rearranged slightly for better "Picard delivering a one-liner to his crew during a one-on-one conversation" experience.

CloudOfNeutronFlux commented:
> * Why a docking door? This is a difficult one to secure (though I recognize why it might be fun to dock here), so people have very rapid access to your bridge. Furthermore I'm uncertain that the shape of your shuttle will allow for much docking from this location. It might also cause some frustration with respect to any rules regarding airlock construction.

It was initially there for Bocadillos and Spirits/Tynes/Searchlights, but I'm not super married to it staying there. Left it during the interior remodel because it recieved no comments. I'll see if I can put something else there instead, to fill the space.


> * Buttons on tables may be a no go. However now that the frames can be deconstructed this could possibly be fine. I'm not entirely sure what the thoughts are here. Technically buttons can be "built" anywhere. I personally feel that it's not a huge deal, but you might receive some sobbing, wailing and gnashing of teeth from the maptainers.

Buttons on tables were shamelessly stolen from the Investigator. Was a bit hesitant at first, but with the button frame PR I think I'm personally okay with them staying there for now.


> * I think you went in a good direction here so far as the space is concerned. However I think that you may now find yourself in an odd position where you have double lounges. As the map continues to develop you may find that you have other uses for this space, as it is rather strategically placed between the bridge and the rapid deployment blast doors.

Agreed, but I won't lie I am kinda attached to the conference room now, so maybe I'll just replace sofas with chairs for the time being for a more serious feel, and then see if something comes up later that'd require this space.


> * I wonder if the main dock couldn't be modified slightly to allow for a cargo conveyor setup. 2x airlocks and 1x conveyor to ease the movement of goods on and off of the shuttle.

Like this? [W - wall, D - dock, C - conveyor]
[W][D][D][C][W]


> * I think the locations of the buttons to actuate the blast doors are a bit awkward to access and could use a second look. It could be that they're functionally more appropriately placed where the current air alarm and fire extinguisher are located for ease of access and use.

Yeah I couldn't decide whether or not to place them there or on the corner, so I tossed a coin instead. I'll play around with them more.


> Medical Bay and Mess Hall

Honestly I'm not super big on either of these myself and I haven't figured out what to do yet.

Medical bays on expedition ships are kinda awkward in general (a bunch of them don't even have a locker, some have soulless empty rooms like Brigand and Dragonfly, and Sprinter and Anchor just have more-or-less stocked basic bays), so it's hard to gauge a good approach here. It does feel like there should be a medbay room at the very least.

DonkElfLawyer commented:
After the rework, I really like it! Very cozy.

CloudOfNeutronFlux commented:
Redid the layout again, and I think I'm much more happy with this one.

Rooms feel better and more defined to me, and thanks to Dusty's dorms trick Raven now has proper (intentionally claustraphobic) bedspace for crew.

I'd love to get some feedback on how the ship handles and whether or not I should add more thrusters or remove a gyro. It feels fine to me but I'm not picky when it comes to this so others' opinions would be welcome.

CloudOfNeutronFlux commented:
> * I think that there may be some utility for a few signs. This air alarm could be an EMS sign to better inform of the space and purpose. The color scheme on the floor is excellent but that was a takeaway that I had with my first impression. Ditto the sign over the power room, while the poster is cute it's not exactly a room indicator.

Good point on the signs, I completely forgot about their existence ~~(because I always remove them in gameplay to fit more shelves ahahahah)~~. I'll see if I can fit them anywhere without cluttering the walls too much.

> Fans

Yeah I rotated those to see if they'd look better visually on this side, but so far it's a potato-potato kind of deal for me, so I'll flip them back in the next iteration.


> Shutters

Same kinda reason here. Wanted to see if they could be made to help imply the shape of the shuttle. I'll wait for some more feedback, see whether this intention is actually seen by people or if this placement just confuses them.
(I do really like how it looks in the conference room in particular, though.)

arimah commented:
This does not constitute a full review of the ship, but I have to just say, _damn_ you're improving fast! I'm impressed. This really does get considerably better with each change. :eyes:

CloudOfNeutronFlux commented:
Added two more rear-facing thrusters and two more lateral thrusters; settled on two gyros. Also removed one SMES unit to cut down the price, but left wiring and a decal to show where you should put one if you want to upgrade to two units.

I think that's about it for the time being, so I'll be waiting for a proper review now, unless something else comes up.

Jakumba commented:
Not useful feedback, but I love the poster placement. Clean hands save lives by the janitorial supplies and bathroom gave me a sensible chuckle.

CloudOfNeutronFlux commented:
Gas tank dispenser moved to EVA access.

Double lights in medbay and captain's quarters replaced with a single led light.

The horrible injustice that was the lack of safety moth posters in engineering has been corrected. Replaced some other static posters with random spawners while I was at it.

Updated the original post to have a better description.

Cheackraze commented:
> A question for the reviewer in advance: do we have to keep mapping cargo pallets if exped rewards aren't coming back? Their design doesn't always play into whatever shuttle they're mapped onto, and they are also a pain to deal with if you want to customize the ship during a round.

for now I would still include them, there may or may not come back into use in the very near future
